salt/rvm/init.sls

```diff
@@ -0,0 +1,32 @@
+{% from 'lib.sls' import create_user %}
+
+rvm:
+  pkgrepo.managed:
+    - ppa: rael-gc/rvm
+  pkg.installed:
+    - name: rvm
+    - require:
+      - pkgrepo: rvm
+
+# A Salt bug breaks support for the apt-managed package.
+rvm salt integration:
+  file.symlink:
+    - name: /usr/local/rvm/bin/rvm
+    - target: /usr/share/rvm/bin/rvm
+    - makedirs: True
+
+{{ create_user('rvm') }}
+
+add rvm user to rvm group:
+  group.present:
+    - name: rvm
+    - addusers:
+      - rvm
+    - require:
+      - user: rvm_user_exists
+
+ruby-{{ pillar.rvm.default_version }}:
+  rvm.installed:
+    - default: True
+    - require:
```

Comment:
Should we add `user: rvm` (and `require: - user: rvm`)? https://docs.saltproject.io/en/latest/ref/states/all/salt.states.rvm.html#salt.states.rvm.installed

Reply:
We don't need to set up an additional RVM user. I've just removed this and confirmed it works happily. 
The `rvm` package sets up a `rvm` group by itself which is all it needs for permissions.